services/crypto_delete_allowance.proto

```diff
@@ -29,34 +37,68 @@ option java_multiple_files = true;
 import "basic_types.proto";
 
 /**
- * Deletes one or more non-fungible approved allowances from an owner's account. This operation
- * will remove the allowances granted to one or more specific non-fungible token serial numbers. Each owner account
- * listed as wiping an allowance must sign the transaction. Hbar and fungible token allowances
- * can be removed by setting the amount to zero in CryptoApproveAllowance.
+ * Delete one or more allowances.<br/>
+ * Given one or more, previously approved, allowances for non-fungible/unique
+ * tokens to be transferred by a spending account from an owning account;
+ * this transaction removes a specified set of those allowances.
+ *
+ * The owner account for each listed allowance MUST sign this transaction.<br/>
+ * Allowances for HBAR cannot be removed with this transaction. The owner
+ * account MUST submit a new `cryptoApproveAllowance` transaction with the
+ * amount set to `0` to "remove" that allowance.<br/>
+ * Allowances for fungible/common tokens cannot be removed with this
+ * transaction. The owner account MUST submit a new `cryptoApproveAllowance`
+ * transaction with the amount set to `0` to "remove" that allowance.<br/>
+ *
+ * ### Block Stream Effects
+ * None
  */
 message CryptoDeleteAllowanceTransactionBody {
     /**
-     * List of non-fungible token allowances to remove.
+     * List of non-fungible/unique token allowances to remove.
+     * <p>
+     * This list MUST NOT be empty.
      */
     repeated NftRemoveAllowance nftAllowances = 2;
+
+    // Note: Field numbers 1 and 3 were very briefly used in early
+    //     versions of this file, but were removed prior to any release
+    //     version. Those field numbers MAY be reused.
 }
```

Comment:
Is it intentional to use this type of comment "//"

Reply:
Yes, it's a comment on code, not API specification.